src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -464,6 +464,24 @@
 short-summary: Manage blob metadata.
 """
 
+helps['storage blob restore'] = """
+type: command
+short-summary: Restore blobs in the specified blob ranges.
+examples:
+  - name: Restore blobs in two specified blob ranges. For examples, (container1/blob1, container2/blob2) and (container2/blob3..container2/blob4).
+    text: az storage blob restore --account-name mystorageaccount -g MyResourceGroup -t 2020-02-27T03:59:59Z -r container1/blob1 container2/blob2 -r container2/blob3 container2/blob4
```

Comment:
`-r container1/blob1 container2/blob2 -r container2/blob3 container2/blob4`  Is it intersection or union in this case?

Reply:
what do you mean for intersection or union? I don't get your point.

Reply:
`-r container1/blob1 container2/blob2` means the bolb is double backed up in a blob range? `-r xxx -r xxx -r xxx...` means backup in multiple blob ranges?

Reply:
no. it is a range.
It means blobls from conrainer1/blob1 ... (container1/blon3) ...to container2/blob2.